binderhub/templates/about.html

```diff
@@ -1,5 +1,9 @@
 {% extends "page.html" %}
 
+{% block head %}
+<meta property="og:description" content="Reproducible, sharable, open, interactive computing environments.">
```

Comment:
Not a huge fan of the "open" here. Do we need it? The previous version had a better "feel" or rhythm to it when reading. Because the rhythm isn't so nice it makes me as a reader think we are just stuffing adjectives in for SEO purposes or so we can tick of buzzwords.

Another point is that I think outside of the bubble of open science not soo many people know what to do with "open" as a description. What is an open computing environment? Anyone can see what I am doing? I think a word that is better is "public" (as in only public repos) but that is confusing too because the env you get isn't public, it is sourced from a public repo.

IMHO the goal should be to let the user know what will happen to them if they click the link (where will I get taken?) and entice them to click it. We don't need to explain what Binder is, people will look that up if they care to know.

This is why I'd go with the three adjectives version.

Reply:
I think three adjectives sounds better too. The reason I'm trying to add "open" is because I see it as one of the main distinguishing factors compared w/ other cloud services. E.g., it'd be somewhat trivial for Colab to add some kind of "share an environment" feature. Where does that leave Binder? I was trying to highlight things that are uniquely Binder Project.

That said, I think it's awkward enough and we don't have consensus enough that it shouldn't be in this PR

Reply:
While "open" is a convenient term, it can have multiple interpretations: open, pseudo-open, open requirements, etc. Some of the distinguishing factors are transparency and inspection/introspection of builds and dependencies. It would be great to at some point break down a reproducible build into the key blocks where inspection is important (requirements, container build, versions).

Reply:
Which is my long winded way of saying that we need to capture and publicize the distinguishing factors. Perhaps not in the title but yes keeping focus on it for those who will care.

Reply:
@willingc I think that's a good point - it actually reminds me [of a recent blog post from tal yarkoni](http://www.talyarkoni.org/blog/2019/07/13/i-hate-open-science/) that you might like if you haven't seen it yet. I agree that we can and should try to dig down to the specifics and why they're useful in BinderHub! Happy to iterate on that in another PR

Reply:
Good points. Investing effort into explaining why Binder is the way it is, why we think that this is a good trade off, etc is a good idea. I keep thinking back to "Binder, so advanced it might contain magic" (advance tech being indistinguishable from magic) and the fact that most magic tricks are actually very simple to understand once they are explained to you. So I am pondering how we can explain Binder like that :) "Binder performs a magic trick" (building your env from nothing), let me show you how it does it. See, there is no magic here, you could do it yourself! *ramblingrambling*